src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -194,6 +195,7 @@ def reference_accesses(self, var_accesses):
         var_accesses.next_location()
 
     def load(self, call, parent=None):
+        # pylint: disable=attribute-defined-outside-init, arguments-differ
```

Comment:
I prefer not to disable those warnings. They are a reminder that this "load" pattern should be refactored.

Reply:
Fair enough, removed.

Reply:
(I just realised that I should remove both disable statements, not only the attribute-defined-outsite-init) ... done :)